src/pages/Send.tsx

```diff
@@ -16,30 +17,57 @@ const Send = () => {
 
     const validationSchema = object().shape({
         amount: string()
+            .required(t('ERROR.IS_REQUIRED', { name: 'Amount' }))
             .matches(/^[0-9]+(\.[0-9]{1,8})?$/, {
                 message: t('ERROR.IS_INVALID', { name: 'Amount' }),
             })
             .test('max-balance', t('ERROR.BALANCE_TOO_LOW'), (value) => {
                 if (!value) return true;
                 const userBalance = primaryWallet?.balance() || 0;
                 return Number(value) <= userBalance;
-            }),
+            })
+            .test(
+                'total-check',
+                t('ERROR.IS_EXCEEDING_BALANCE', { name: 'fee + amount' }),
+                (value) => {
+                    if (!value || !formik.values.fee) return true;
+                    const userBalance = primaryWallet?.balance() || 0;
+                    const sum: number = Number(value) + Number(formik.values.fee);
+                    return sum <= userBalance;
+                },
+            )
+            .trim(),
         memo: string().max(255, t('ERROR.IS_TOO_LONG', { name: 'Memo' })),
+        fee: string()
+            .required(t('ERROR.IS_REQUIRED', { name: 'Fee' }))
+            .matches(/^[0-9]+(\.[0-9]{1,8})?$/, {
```

Comment:
maybe move the regex out to a constant with a descriptive name, as it's used twice